### ISSUE TITLE
COPY failing in 0.10.4

### ISSUE THREAD

**Issue by roveo** [vertica/vertica-python#377]
Query:

```
   BEGIN;
        TRUNCATE TABLE ***;
        COPY ***
        (
            ***
        )
        FROM STDIN
        PARSER FCsvParser()
        ABORT ON ERROR
        REJECTED DATA AS TABLE ***;
    COMMIT;
```

I know that BEGIN and COMMIT make no sense here, but it worked in 0.10.3

Error from Airflow logs:

```
[2020-05-22 13:48:56,264] {{taskinstance.py:1058}} ERROR - Received unexpected message type: CommandComplete. Expected type: CopyInResponse
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 930, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/airflow/pikprodata/pikprodata/operators/postgres_to_vertica_operator.py", line 162, in execute
    cur.copy(self.sql, self.extract_func())
  File "/usr/local/lib/python3.7/site-packages/vertica_python/vertica/cursor.py", line 403, in copy
    self._copy_error_handler(sql))
  File "/usr/local/lib/python3.7/site-packages/vertica_python/vertica/connection.py", line 637, in read_expected_message
    raise errors.MessageError(msg)
vertica_python.errors.MessageError: Received unexpected message type: CommandComplete. Expected type: CopyInResponse
```

Version: `Vertica Analytic Database v9.3.0-0`

**Comment by sitingren** [comment]
@roveo Thanks for your report. We are making changes for COPY/COPY LOCAL protocols in v0.10.4 and the next future release. I am sorry for any inconveniences caused by this change.

Looks like you passed a multi-statement query to `Cursor.copy()` and COPY was not the first statement in the query. Could you please try splitting your query and make COPY as the first statement in the query to `Cursor.copy()`?

**Comment by sitingren** [comment]
This issue should be fixed in v0.11.0. Please upgrade the package and try again.
